main.cpp

```diff
@@ -648,6 +796,20 @@ struct JumboJet
     bool fly(bool safetyInspection, double gust);
     void carryCargo(int maxWeightPermitLuggagePerPassenger);
     std::string carryFuel(int);
+    void increaseAirportProfit(int airportProfit, int flightsPerDay)
+    {
+        auto fpd = flightsPerDay;
+        while (flightsPerDay < 136)
+        {
+            for (fpd = 1; fpd < 3; ++fpd)
+            {
+                airportProfit += 100;
+                std::cout << "Our airport is growing." << std::endl;
+            }
+        break;
+        }
```

Comment:
remove the `break`, because it prevents the `while` loop from looping.
- [ ] revise